test/nuttx-config.js

```diff
@@ -0,0 +1,122 @@
+/* Copyright 2016 Samsung Electronics Co., Ltd.
```

Comment:
@yichoi I renamed configure file as you mentioned.  Also, I made the driver that selects the testset by given argument. Please check this out!